hooks/post/30-upgrade.rb

```diff
@@ -178,32 +74,7 @@ def fail_and_exit(message)
     upgrade_step :restart_services, :run_always => true
 
     if Kafo::Helpers.module_enabled?(@kafo, 'katello')
```

Comment:
This this be generalized to just `module_enabled?('foreman')`? It should use the [kafo hooks DSL](https://github.com/theforeman/kafo#hooks) and since the rake task is in core foreman it means we can later easily port it to Foreman.

Reply:
works4me!